docs/cli.md

```diff
@@ -127,8 +131,36 @@ lhci collect --numberOfRuns=5 --url=https://example.com
 lhci collect --start-server-command="yarn serve" --url=http://localhost:8080/
 lhci collect --staticDistDir=./dist
 lhci collect --url=https://example-1.com --url=https://example-2.com
+lhci collect --start-server-command="yarn serve" --url=http://localhost:8080/ --puppeteer-script=./path/to/login-with-puppeteer.js
 ```
 
+#### Using Puppeteer Scripts
+
+When running Lighthouse CI on a page with behind authentication, you'll need to authorize the browser that Lighthouse CI will be using. While there are [several](./configuration.md#page-behind-authentication) different [options](https://github.com/GoogleChrome/lighthouse/blob/v5.6.0/docs/authenticated-pages.md) to accomplish this, `--puppeteer-script` is one of the most flexible and convenient.
+
+`--puppeteer-script` accepts a path to a JavaScript file that exports a function that will be invoked by Lighthouse CI before navigating, e.g. a script that will login to your site.
+
+**Example `puppeteer-script.js`**
+
+```js
+/**
+ * @param {puppeteer.Browser} browser
+ * @param {{url: string, options: LHCI.CollectOptions}} context
```

Comment:
I don't think this type is defined anywhere. is it `{url:string, options: LHCI.Options}`?

Reply:
yeah it's not it was meant to just be user friendly "here are the collect options" but i'll use the real one :)